tests/templates/instance.tfvars.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tenancy_ocid": "ocid1.tenancy.oc1..aaaaaaaaojlj67jpdjvx7wszzrnwnooss3xxt66oldeyt2x35a6gthqdfr4q",
```

Comment:
You really should not be committing this to GitHub

Reply:
applied